egs2/voxceleb/spk1/conf/train_RawNet3.yaml

```diff
@@ -0,0 +1,35 @@
+# RawNet3 reproduce recipe configuration.
+encoder: rawnet3
+#encoder_conf:  # uncomment for customized arguments
```

Comment:
It is better to add some of the key arguments of `encoder` here (e.g., the dimensions), even though they have the default values.